src/pandas_profiling/visualisation/plot.py

```diff
@@ -223,7 +262,7 @@ def scatter_complex(config: Settings, series: pd.Series) -> str:
     plt.ylabel("Imaginary")
     plt.xlabel("Real")
 
-    color = config.html.style.primary_color
+    color = config.html.style.primary_colors[0]
```

Comment:
This doesn't seem to be required since the `Style` class has the `primary_color` property anyway..?

Reply:
We should deprecate `primary_color`. Not intending to make this feature breaking backwards compatibility. Changes in preparation of removal.